### ISSUE TITLE
Nuget

### ISSUE THREAD

**Issue by dalmat501** [radimitrov/CSharpShellApp#148]
Sorry but the new version is a disaster, the nuget packages don't running, and you can't add new packages from a list like the old version. 

Please 🙏 fix the bug

**Comment by AmiSimon** [comment]
I think you have to download the package from the "Browse" section. Next add the package to project from the tab where there is all the packages you downloaded. Next you can use it.

**Comment by dalmat501** [comment]
Ok👌.Thanks 😊 for all

**Comment by radimitrov** [comment]
True, what @AmiSimon is saying is correct. I've added a checkbox that makes it more apparent.
However I also found a problem that prevented packages from being recognized on Android 5. It's fixed.

**Comment by dalmat501** [comment]
Hi radimitrov, all the problems were resolved. Moreover I bought a suscription for AspNet in your application and all function correctly. Anyway if I try make an application to call API under aspnetcore don't recognize some types like WebApplication.CreateBuilder(). I realize that the package using Microsoft.AspNetCore.Builder not include it. When I try download from nuget don't appear. Is a error of the company or has migrated to another? Thanks ,😅
